pkg/tasks/alertmanager_user_workload.go

```diff
@@ -47,6 +48,7 @@ func (t *AlertmanagerUserWorkloadTask) Run(ctx context.Context) error {
 		return t.create(ctx)
 	}
 
+	klog.V(3).Infof("UWM alertmanager is disabled, existing related resources are to be destroyed.")
```

Comment:
UWM alertmanager is disabled by default, maybe we don't need to check if it's disabled
https://github.com/openshift/cluster-monitoring-operator/blob/master/pkg/manifests/types.go#L566-L574

Reply:
The log is just reflecting the logic; the code checking if it's enabled/disabled